beacon-chain/sync/validate_attester_slashing_test.go

```diff
@@ -163,11 +163,12 @@ func TestValidateAttesterSlashing_ValidOldSlashing(t *testing.T) {
 func TestValidateAttesterSlashing_InvalidSlashing_WithdrawableEpoch(t *testing.T) {
 	p := p2ptest.NewTestP2P(t)
 	ctx := context.Background()
-
+	params.BeaconConfig().MinSlashingWithdrawableDelay = 0
+	params.BeaconConfig().MinValidatorWithdrawabilityDelay = 0
```

Comment:
I'm little worried about overriding fields at a global object like this, because concurrent test jobs can share same memory address.

Maybe the best practice to modify the config setting is like following.  How do you think of this?

```go
	cfg := params.BeaconConfig().Copy()
        cfg.MinSlashingWithdrawableDelay = 0
	cfg.MinValidatorWithdrawabilityDelay = 0
	params.OverrideBeaconConfig(cfg)
```

Reply:
I agree, but I'm curious if ₩params.OverrideBeaconConfig(cfg)` ultimately makes it meaningless.